README.md

```diff
@@ -16,82 +20,811 @@ yarn add gotrue-js
-  error => console.log("Failed to get list of users: %o", error)
-)
-
-const user = { id: 1234 }
-admin.getUser(user).then( 
-  user => console.log("User object: %o", user),
-  error => console.log("Failed to retrieve user: %o", error)
-)
-
-// Admins updating email does not require a confirmation step
-const attributes = {email: newEmail, password: newPassword}
-admin.updateUser(user, attributes).then(
-  user => console.log("Updated user: %o", user),
-  error => console.log("Failed to update user: %o", error)
-)
-
-admin.createUser(email, password, attributes = {}).then(
-  user => console.log("Created user user: %o", user),
-  error => console.log("Failed to create user: %o", error)
-)
-
-admin.deleteUser(user).then(
-  user => console.log("Deleted user user: %o", user),
-  error => console.log("Failed to delete user: %o", error)
-)
+import GoTrue from "gotrue-js";
+
+// Instantiate the GoTrue auth client with an optional configuration
+
+auth = new GoTrue({
+  APIUrl: "https://<your domain name>/.netlify/identity",
+  audience: "",
+  setCookie: false
+});
+```
+
+### GoTrue configuration
+
+APIUrl: The absolute path of the GoTrue endpoint. To find the `APIUrl`, go to `Identity` page of your Netlify site dashboard.
+
+audience(optional): `audience` is one of the pre-defined [JWT payload](https://tools.ietf.org/html/rfc7519#section-4.1.3) claims. It's an optional attribute which is set to be empty by default. If you were hosting your own identity service and wanted to support [multitenancy](https://en.wikipedia.org/wiki/Multitenancy), you would need `audience` to separate the users.
+
+setCookie(optional): set to be `false` by default. If you wish to use the `remember me` functionality, set the value to be `true`.
+
+### Error handling
+
+If an error occurs during the request, the promise may be rejected with an Error, `HTTPError`, `TextHTTPError`, or `JSONHTTPError`. See [micro-api-client-lib error types](https://github.com/netlify/micro-api-client-lib#class-httperror-extends-error).
+
+## Authentication methods
+
+### Create a new user
+
+Create a new user with the specified email and password
+
+```js
+auth.signup(email, password);
+```
+
+Example usage:
+
+```js
+auth
+  .signup(email, password)
+  .then(response => console.log("Confirmation email sent", response))
+  .catch(error => console.log("It's an error", error));
+```
+
+Example response object:
+
+```js
+{
+  id: 'example-id',
+  aud: '',
+  role: '',
+  email: 'example@example.com',
+  confirmation_sent_at: '2018-04-27T22:36:59.636416916Z',
+  app_metadata: { provider: 'email' },
+  user_metadata: null,
+  created_at: '2018-04-27T22:36:59.632133283Z',
+  updated_at: '2018-04-27T22:37:00.061039863Z'
+}
+```
+
+Also, make sure the `Registration preferences` under `Identity settings` in your Netlify dashboard are set to `Open`.
+
+[registration preferences](https://github.com/netlify/gotrue-js/blob/update-readme/src/images/identity-settings-registration.png)
```

Comment:
Move this into a link in the preceding sentence.

Reply:
Oh it's supposed to be rendered as an image, I'll fix it

Reply:
Oh ok, well in that case!